Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteProjectController.java

```diff
@@ -215,5 +224,104 @@ public String visualizeAllNetworks(@PathVariable("projectUnixName") String proje
 
 		return "sites/networks/visualize";
 	}
+	
+	public static JSONArray sortByValue(Map<String, Integer> map, Map<String, String[]> map2) throws JSONException {

```

Comment:
A few things:
1. formatting
2. why is this a static method?
3. I think you can delete this methods. See later comments.